modules/cassandra/src/test/java/org/testcontainers/containers/CassandraContainerTest.java

```diff
@@ -30,8 +28,8 @@ public void testSimple() {
         try (CassandraContainer<?> cassandraContainer = new CassandraContainer<>(CASSANDRA_IMAGE)) {
             cassandraContainer.start();
             ResultSet resultSet = performQuery(cassandraContainer, BASIC_QUERY);
-            assertTrue("Query was not applied", resultSet.wasApplied());
-            assertNotNull("Result set has no release_version", resultSet.one().getString(0));
+            assertThat(resultSet.wasApplied()).as("Query was not applied").isTrue();
+            assertThat(resultSet.one().getString(0)).as("Result set has no release_version").isNotNull();
```

Comment:
```suggestion
            assertThat(resultSet.wasApplied()).as("Query was").isTrue();
            assertThat(resultSet.one().getString(0)).as("Result set has release_version").isNotNull();
```

In Assertj, the description in the `as()` has to be a valid case to make sense, or? Consider this comment for the whole class.